src/scripts/main.js

```diff
@@ -1,8 +1,30 @@
 'use strict';
 
 const people = require('./lib/people');
+const table = document.querySelector('tbody');
 
 // eslint-disable-next-line no-console
-console.log(people); // you can remove it
-
+console.log(people);
 // write your code here
+
+for (const person of people) {
+  let sex = '';
+
+  if (person.sex === 'm') {
+    sex = 'Male';
+  } else {
+    sex = 'Female';
+  }
```

Comment:
```suggestion
```

You can use the ternary operator directly in your `insertAdjacentHTML` method instead of this `if...else`